tests/config.c

```diff
@@ -60,6 +60,7 @@ config_set_defaults(master_config *cfg)
       .use_stats                = FALSE,
       .reclaim_threshold        = UINT64_MAX,
       .verbose_logging_enabled  = FALSE,
+      .verbose_progress         = FALSE,
```

Comment:
[ Intentionally chose `verbose_progress` rather than `verbose_progress_enabled` as for a boolean variable, I felt, the term `_enabled` is redundant. ]